src/overlays/actors/ovl_En_Door/z_en_door.c

```diff
@@ -27,29 +40,347 @@ const ActorInit En_Door_InitVars = {
     (ActorFunc)EnDoor_Update,
     (ActorFunc)EnDoor_Draw,
 };
-*/
+
+/**
+ * Controls which object and display lists to use in a given scene
+ */
+EnDoorInfo D_809FCEA0[] = {
```

Comment:
you can make the data `static`

Reply:
Not all data can be made `static` if it is used in the `NON_MATCHING`s as the version of asm-processor we currently use does not deal with static symbols

Reply:
But it can be static now.